### ISSUE TITLE
`youtube-dl -U` fails to update (update url is currently 404)

### ISSUE THREAD

**Issue by alexandr-** [rg3/youtube-dl#18079]
currently the [url "https://rg3.github.io/youtube-dl/update/"](https://github.com/rg3/youtube-dl/blob/master/youtube_dl/update.py#L34) will return a 404 so that the update mechanism will fail.

**Comment by alexandr-** [comment]
ok, let me rephrase: an older version i had installed failed:
" :~$ youtube-dl -U
ERROR: can't find the current version. Please try again later." 

and https://github.com/rg3/youtube-dl/blob/master/youtube_dl/update.py#L34 still mentions that url?

**Comment by dstftw** [comment]
Most likely your python is too old and does not support TLS 1.2. Updating on such python versions won't work.
Again: https://rg3.github.io/youtube-dl/update/ is not [used](https://github.com/rg3/youtube-dl/blob/c0345b825f8758571a8de871ab9349c46b062fc0/youtube_dl/update.py#L45) [directly](https://github.com/rg3/youtube-dl/blob/c0345b825f8758571a8de871ab9349c46b062fc0/youtube_dl/update.py#L45) for updating.
